examples/e2e/provider.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from flask import Flask, abort, jsonify, request
+
+fakedb = {}
+
+app = Flask(__name__)
+
+
+@app.route('/_pact/provider_states', methods=['POST'])
```

Comment:
Is there a way you can dynamically add this endpoint during the verification test so it's clear this endpoint doesn't need to be present in a deployed environment? This might confuse some people coming in - we want to make it clear that the states endpoint should only be added for testing purposes.

Reply:
I've been moving stuff from the e2e folder into examples (and from other sources) to make it a bit simpler for newbies. The current e2e tests are for local and weren't totally clear how to run. But I'll keep chipping away. I think I can do what you want via some kinda mix in in python but I'll look into it.

Reply:
I've created a first pass at this. There is in the root folder a pact_provider file that imports from the provider plus decorating with extra pact endpoints